plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/ThorntailExtension.java

```diff
@@ -256,11 +261,50 @@ public boolean isHollow() {
     @Override
     public Map<DependencyDescriptor, Set<DependencyDescriptor>> getDependencies() {
         if (dependencyMap == null) {
-            dependencyMap = GradleDependencyResolutionHelper.determineProjectDependencies(project, "default");
+            dependencyMap = GradleDependencyResolutionHelper.determineProjectDependencies(project, "runtimeClasspath", false);
         }
         return dependencyMap;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Map<DependencyDescriptor, Set<DependencyDescriptor>> getTestDependencies() {
```

Comment:
Instead of polluting the existing `getDependencies` method, I have included a separate method for `test` scoped dependencies.